src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -541,20 +511,16 @@ const deletedTodo = await API.graphql({
 
 </BlockSwitcher>
 
-Only an `id` is needed.
-
 <Callout>
-Join table records must be deleted prior to deleting the associated records. For example, for a many-to-many relationship between Posts and Tags, delete the PostTags join record prior to deleting a Post or Tag.
-</Callout>
 
-#### Troubleshooting
+__Note:__ Join table records must be deleted prior to deleting the associated records. For example, for a many-to-many relationship between Posts and Tags, delete the PostTags join record prior to deleting a Post or Tag.
+
+</Callout>
 
-[comment]: # (EXPANDER SECTION to provide tips on how to avoid common errors such as: unauthorized errors and cross-platform gotchas. Each error should have a header with descriptive title to be included on a primary troubleshooting guide as well. Added Auth info from https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/graphqlapi/js/mutate-data.mdx - not sure if this one should be for troubleshooting or general info added outside the expander.)
-[question]: #(RENE, is the following section part of Troubleshooting? The following section seems to be more focused on offroading than troubleshooting.) 
 
-##### Custom authorization mode
+<Accordion title='Custom authorization mode to mutate data' headingLevel='4' eyebrow='Learn more'>
```

Comment:
Kept the Learn more eyebrow text instead of troubleshooting on the expander below b/c this reads as more of an option. We originally had a note to provide troubleshooting for unauthorized errors and cross-platform gotchas - this does not read to me like it covers those errors. Additionally I think the intro within this expander could add an explanation of why a dev may want to override this